kubeflow/jupyter/tests/jupyter_test.py

```diff
@@ -97,7 +97,9 @@ def send_request(*args, **kwargs):
   return r
 
 
-def test_jupyter(env, namespace):
+def test_jupyter(record_xml_attribute, env, namespace):
```

Comment:
what is the type of record_xml_attribute?

Reply:
Its a test fixture created by pytest.